### ISSUE TITLE
BBB remove eeprom patch

### ISSUE THREAD

**Issue by HarelAshwal** [RobertCNelson/Bootloader-Builder#9]
LK"I

Hi ,
I removed the EEPROM on my BBB (Rev C). 
Than the board isn't starting (I also didn't get any messages from the UART 0 debug port...)
I tried to apply this patch and recompile u-boot:
0002-NFM-Production-eeprom-assume-device-is-BeagleBone-Bl.patch 

But it's still doesn't work...
Is this patch should resolve my issue? or it's not enough?

Harel

**Comment by HarelAshwal** [comment]
LK"I

In addition,
Will the above patch work if i'll also issue this patch : 

https://patches.linaro.org/patch/94800/

It says it makes all the EEPROM reading return 0's

Harel

**Comment by RobertCNelson** [comment]
Sorry lack of a physical eeprom has not been tested...

**Comment by HarelAshwal** [comment]
LK"I

Any plans to try it out?
It would probably be very useful for the community... :)

Harel

**Comment by RobertCNelson** [comment]
Nah, that'll create other issues. I won't help support boards that can't 'identify' themselves on boot up.

Regards,

**Comment by HarelAshwal** [comment]
LK"I

Okay, I've create a patch to make this work.
Should I send it over to you? Maybe someone will find it helpful... ?

Harel
